weldx/tests/transformations/test_cs_manager.py

```diff
@@ -737,6 +737,18 @@ def test_comparison_wrong_type():
     assert (csm != 4) is True
 
 
+def test_comparison_data():
+    csm1 = tf.CoordinateSystemManager("root", "csm")
+    csm2 = tf.CoordinateSystemManager("root", "csm")
+    data = np.arange(9).reshape((3, 3))
```

Comment:
```suggestion
    data = np.arange(12).reshape((3, 4))
```
Not really relevant here (so no need to apply the suggestion), but symmetrical shapes in tests, tend to be dangerous. They can accidentally pass tests they shouldn't pass.

Reply:
Now we receive this error:  ValueError: conflicting sizes for dimension 'c': length 4 on the data but length 3 on coordinate 'c'

Was this intended @vhirtham?

Reply:
Not really intended. Just forgot that the last dimension needs to be of length 3 since it gets converted to a `SpatialData` IIRC. So changing the shape to `(4, 3)` should fix this